Dshop/apps/products_catalogue/tests/test_api_products.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from django.urls import reverse
+from apps.products_catalogue.models import Category, Product
+# pylama:ignore=W0404, W0611
+from apps.users.conftest import api_client, login_url, login_data, user_instance, user_instance_token
+
+
+@pytest.fixture
+def create_category():
+    return Category.objects.create(name='Test Category', is_active=True)
+
+
+@pytest.fixture
+def create_product():
+    category = Category.objects.create(name='Test Category', is_active=True)
+    return Product.objects.create(
+        name="main product",
+        category=category,
+        price=11,
+        short_description="short desc",
+        full_description="full_description"
+    )
+
+
+@pytest.mark.django_db
+def test_access_protected_resource(api_client, user_instance_token):
```

Comment:
Please add one or two products here so we can see products structure in tests

Reply:
Would be good to add active and not active products to see if is_active flag is followed.

Reply:
done